packages/components/Footer.tsx

```diff
@@ -9,11 +9,12 @@ import { Logo } from "@packages/components/icons";
 
 import Section from "./Section";
 import SocialIconLinks from "./SocialIconLinks";
+import { classes } from "@packages/utils/classes";
 
 export default function Footer() {
-  const bgColor = "#0a1523";
+  const bgColor = "bg-neutral-950";
   return (
-    <Section p="1.5rem" bg={bgColor} alignItems="center">
+    <Section className={classes(bgColor, 'px-6 py-6 items-center')}>
```

Comment:
```suggestion
    <Section className={classes(bgColor, 'p-6 items-center')}>
```

Reply:
In this case, what makes it impossible is that our classes function does not recognize `p` while also handling `py`. This creates a conflict because the `py` defined in the parent is also accepted, resulting in duplicated padding.